airflow/settings.py

```diff
@@ -173,10 +173,21 @@ def configure_orm(disable_connection_pool=False):
         except conf.AirflowConfigException:
             pool_recycle = 1800
 
+        # Check connection at the start of each connection pool checkout.
+        # Typically, this is a simple statement like “SELECT 1”, but may also make use
+        # of some DBAPI-specific method to test the connection for liveness.
+        # More information here:
+        # https://docs.sqlalchemy.org/en/13/core/pooling.html#disconnect-handling-pessimistic
+        try:
+            pool_pre_ping = conf.getboolean('core', 'SQL_ALCHEMY_POOL_PRE_PING')
```

Comment:
```suggestion
            pool_pre_ping = conf.getboolean('core', 'SQL_ALCHEMY_POOL_PRE_PING', fallback=False)
```

And then we don't need the try/except.